### DISCUSSION TITLE
Using BREAK to start new chunk?

### DISCUSSION THREAD

**Discussion by Rabbit-kun** [lllyasviel/stable-diffusion-webui-forge#2243] [Q&A]
I'm moving from A1111 to Forge full time. On A1111 I can use BREAK to finish off the chunk I'm on and start a new one. Forge doesn't appear to do that, the token counter in the corner doesn't reflect it at least.

Is there a different command to use in Forge? Or does Forge just not recognize it?

(I tried to search for an answer to this but was unsuccessful)

**Comment by jswag245** [comment]
It should be the same command. I am able to use it with no issues.

**Reply by Rabbit-kun** [reply]
Thanks, it's good to know it _should_ work, now I just need to find out why it won't.

I deleted the entire folder Forge was located in and reinstalled it from there. At least I know it can be done.

Thanks again mate!

**Comment by AnabelleKM** [comment]
What would be the correct way to use the BREAK command?

1) tag1, tag2, BREAK, tag 3, tag4
2) tag1, tag2 BREAK tag3, tag4
3) tag1, tag2
BREAK
tag3, tag4

Or does a line break even act the same as the BREAK command? e.g:
tag1, tag2,
tag3, tag4,

**Reply by jswag245** [reply]
I believe that all 3 ways are acceptable, but I use 3 to help organize my prompt. I don't think that line breaks work the same as the command

**Reply by AnabelleKM** [reply]
I use 3. as well. But haven't really tested whether that does make a difference or not. But yes, it keeps prompting organized. ;-)

**Comment by SvenErik1968** [comment]
The [wiki on A1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#break-keyword) says:

> Adding a `BREAK` keyword (must be uppercase) fills the current chunks with padding characters. Adding more text after `BREAK` text will start a new chunk.

**Comment by chaewai** [comment]
Am I correct that this and AND / OR only work with SD, not Flux? Anyone know if it is viable to implement for Flux?
